@fraserg-arista  there is one failing test can you check 

=========================== short test summary info ============================
FAILED tests/liquid_cooling_base_test.py::TestLiquidCoolingBase::test_get_leak_sensor_out_of_range
================== 1 failed, 987 passed, 8 warnings in 44.51s ==================

--- a/sonic_platform_base/liquid_cooling_base.py
+++ b/sonic_platform_base/liquid_cooling_base.py
@@ -129,6 +129,7 @@ class LiquidCoolingBase(device_base.DeviceBase):
     """
 
     def __init__(self,
+                 leakage_sensors_num: int = 0,
                  leakage_sensors_list: List[LeakageSensorBase] = [],
                  *,
                  profiles: List[LeakSensorProfileBase] = []):

--- a/tests/liquid_cooling_base_test.py
+++ b/tests/liquid_cooling_base_test.py
@@ -277,7 +277,8 @@ class TestLiquidCoolingBase():
         '''
         common = {"type": "rope", "location": "chassis",
                        "severity": LeakSeverity.MINOR}
-        liquid_cooling = LiquidCoolingBase([LeakageSensorBase("Sensor1", **common), 
+        liquid_cooling = LiquidCoolingBase(0,
+                                           [LeakageSensorBase("Sensor1", **common),
                                             LeakageSensorBase("Sensor2", **common), 
                                             LeakageSensorBase("Sensor3", **common)])
 

